### PR TITLE
[Minor] Bugfix + API improvement in lie group.

### PR DESCRIPTION
**Pull request by jmirabel**

### CODE REVIEW

**Review by jcarpent — APPROVED** [review]
Thanks for the PR.

File: src/multibody/liegroup/liegroup-variant-visitors.hpp [hunk 1]
```diff
@@ -155,23 +155,21 @@ namespace pinocchio
                    const Eigen::MatrixBase<JacobianOut_t> & J,
                    const ArgumentPosition arg);
 
-  template<typename LieGroupCollection, class ConfigL_t, class ConfigR_t, class JacobianIn_t, class JacobianOut_t>
+  template<ArgumentPosition arg, typename LieGroupCollection, class ConfigL_t, class ConfigR_t, class JacobianIn_t, class JacobianOut_t>
```
Comment (jcarpent) [inline src/multibody/liegroup/liegroup-variant-visitors.hpp:158]: Do you think that they might be some impact on the end-user by changing the API?

Comment (jmirabel) [reply src/multibody/liegroup/liegroup-variant-visitors.hpp:158]: It shouldn't. I only made the declaration fit with the definition.

Note that I couldn't find the definition of the previously declared function but I may have missed it.

**Comment by jmirabel** [comment]
No idea why gitlab-ci failed.

**Pull request merged**